src/cc_catalog_airflow/dags/provider_api_scripts/brooklyn_museum.py

```diff
@@ -16,18 +16,18 @@
 
 logger = logging.getLogger(__name__)
 
-DELAY       = 3.0 #time delay (in seconds)
-RETRIES     = 3
-FILE        = 'brooklynmuseum_{}.tsv'.format(int(time.time()))
-API_KEY     = os.getenv('BROOKLYN_MUSEUM_API_KEY')
-PROVIDER    = 'brooklynmuseum'
-LIMIT       = 10
-ENDPOINT    = 'https://www.brooklynmuseum.org/api/v2/'
+DELAY = 3.0  # time delay (in seconds)
+RETRIES = 3
+FILE = 'brooklynmuseum_{}.tsv'.format(int(time.time()))
```

Comment:
output file path is not necessary  since the default path is handled by ImageStore